entities/schema/errors.go

```diff
@@ -0,0 +1,13 @@
+package schema
+
+const (
+	ErrorNoSuchClass    string = "no such class with name '%s' found in the schema. Check your schema files for which classes are available"
+	ErrorNoSuchProperty string = "no such prop with name '%s' found in class '%s' in the schema. Check your schema files for which properties in this class are available"
+	ErrorNoSuchDatatype string = "given value-DataType does not exist."
+)
+
+type ErrRefToNonexistentClass struct{}
```

Comment:
Instead of using a struct I would simply define ErrRefToNonexistentClass as:
`var ErrRefToNonexistentClass = errors.New("reference property to nonexistent class")`

and use: `if errors.Is(err, schema.ErrRefToNonexistentClass){ ...`

Reply:
Sure, but what is the benefit in doing so?